lib/sorcery/controller.rb

```diff
@@ -98,7 +98,7 @@ def current_user=(user)
       # used when a user tries to access a page while logged out, is asked to login,
       # and we want to return him back to the page he originally wanted.
       def redirect_back_or_to(url, flash_hash = {})
-        redirect_to(session[:return_to_url] || url, flash: flash_hash)
+        redirect_to(session[:return_to_url] || url, flash: flash_hash, allow_other_host: true)
```

Comment:
The redirect_back_or_to method, unlike what's described in https://github.com/Sorcery/sorcery/pull/318, might accept user input depending on how it's used. Therefore, it would be more secure and preferable to not set allow_other_host to true by default, but rather, to provide it as an argument.